schemas/2018-01-10/Microsoft.RecoveryServices.SiteRecovery.json

```diff
@@ -0,0 +1,2806 @@
+{
```

Comment:
Please could you remove `schemas/2018-01-10/Microsoft.RecoveryServices.json` as this file duplicates those resource types.